### ISSUE TITLE
The file includes the moc file "QNavitWidget.moc", but does not contain a Q_OBJECT, Q_GADGET or Q_NAMESPACE macro

### ISSUE THREAD

**Issue by yurivict** [navit-gps/navit#780]
The build fails on FreeBSD:
```
[9/2390] cd /usr/ports/misc/navit/work/.build/navit/graphics/qt5 && /usr/local/bin/cmake -E cmake_autogen /usr/ports/misc/navit/work/.build/navit/graphics/qt5/CMakeFiles/graphics_qt5_autogen.dir/AutogenInfo.cmake Release
AutoMoc warning
---------------
  "/usr/ports/misc/navit/work/navit-0.5.3-556-gb2205d39/navit/graphics/qt5/QNavitWidget.cpp"
The file includes the moc file "QNavitWidget.moc", but does not contain a Q_OBJECT, Q_GADGET or Q_NAMESPACE macro.

AutoMoc warning
---------------
  "/usr/ports/misc/navit/work/navit-0.5.3-556-gb2205d39/navit/graphics/qt5/event_qt5.cpp"
The file includes the moc file "event_qt5.moc", but does not contain a Q_OBJECT, Q_GADGET or Q_NAMESPACE macro.

AutoMoc: /usr/ports/misc/navit/work/navit-0.5.3-556-gb2205d39/navit/graphics/qt5/QNavitWidget.cpp:0: Note: No relevant classes found. No output generated.
AutoMoc: /usr/ports/misc/navit/work/navit-0.5.3-556-gb2205d39/navit/graphics/qt5/event_qt5.cpp:0: Note: No relevant classes found. No output generated.
ninja: build stopped: subcommand failed.
```

**Comment by metalstrolch** [comment]
Should be fixed by #798

**Comment by metalstrolch** [comment]
Closing this as fixed with #798
